capabilities/eventing.md

```diff
@@ -0,0 +1,132 @@
+---
+displayName: Eventing
+epicsLabels:
+  - area/eventing
+  
+---
+
+## Scope
+
+* Enable asynchronous processing
+* Integrate external solutions with Kyma using an event-driven architecture.
+* Enable loose coupling of applications deployed on Kyma.
+
+
+## Vision
+
+### Pluggability
+* Enable customers to use a underlying messaging implementation that best fit their requirements.
+
+![](assets/pluggable.png)
+
+**Value Addition**
+* Decouple workload from the management plane
+* Flexiblity
+  * Enable customers to choose the solution when it comes to volume, latency, scalability needs
+  * Allows a customer to use the solution already existing in her ecosystem.
+
+### System metrics
+
```

Comment:
Maybe we can add an introductory sentence such as:

`Use the following metrics to measure health and performance of the system:`